build-site.sh

```diff
@@ -54,8 +54,11 @@ check_environment_variables() {
   if [ -z "$JEKYLL_ENV" ]; then
     export JEKYLL_ENV=staging
   fi
+  if [ -z "$JEKYLL_PORT" ]; then
+  JEKYLL_PORT=4000
```

Comment:
```suggestion
    JEKYLL_PORT=4000
```
add same indentation as rest of file